api/config/config.go

```diff
@@ -47,6 +47,8 @@ type Config struct {
 	UI                        UIConfig
 	StandardTransformerConfig StandardTransformerConfig
 	MlflowConfig              MlflowConfig
+
+	EnablePprof bool `envconfig:"ENABLE_PPROF" default:"false"`
```

Comment:
https://github.com/gojek/merlin/pull/164/files#r678884389 To reduce bloat in our configuration.